src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/dto/PrisonerDetailRequest.kt

```diff
@@ -51,6 +51,9 @@ data class PrisonerDetailRequest(
   )
   val prisonIds: List<String>? = emptyList(),
 
+  @Schema(description = "Include aliases in search", example = "true", required = false, defaultValue = "true")
+  val includeAliases: Boolean = true,
```

Comment:
Defaulted to **true** to not break existing functionality which searches across aliases.